services/galley/test/integration/API/Teams.hs

```diff
@@ -187,20 +182,12 @@ timeout = 3 # Second
 
 testCreateTeam :: TestM ()
 testCreateTeam = do
-  c <- view tsCannon
   owner <- Util.randomUser
-  WS.bracketR c owner $ \wsOwner -> do
-    tid <- Util.createNonBindingTeam "foo" owner []
-    team <- Util.getTeam owner tid
-    assertQueueEmpty
-    liftIO $ do
-      assertEqual "owner" owner (team ^. teamCreator)
-      eventChecks <- WS.awaitMatch timeout wsOwner $ \notif -> do
-        ntfTransient notif @?= False
-        let e = List1.head (WS.unpackPayload notif)
-        e ^. eventTeam @?= tid
-        e ^. eventData @?= EdTeamCreate team
-      void $ WS.assertSuccess eventChecks
+  tid <- Util.createBindingTeamInternal "foo" owner
```

Comment:
you should keep the code that checks for the team creation events.  (unless those events are only triggered for non-binding teams?)

Reply:
We check for the actived event, there is no creation event. And we also assert that the queue is empty.